data/indigenous.txt

```diff
@@ -8,6 +8,49 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
+ship "Subsidurial"
+	noun "creature"
+	sprite "ship/subsidurial"
+	thumbnail "thumbnail/subsidurial"
+	"never disabled"
+	attributes
+		category "Light Freighter"
+		"hull" 11600
+		"required crew" 0
+		"bunks" 0
```

Comment:
Zero crew ships *must* have the automaton attribute, as otherwise the game forces at least one crew onto the ship.

Reply:
Fixed.